book/history.md

```diff
@@ -443,12 +442,12 @@ The web has come a long way! But one thing seems clear: it isn't done yet.
 Exercises
 =========
 
-*What comes next*: Based on what you learned about how the web came about and
+*ii*-1 *What comes next?* Based on what you learned about how the web came about and
```

Comment:
This is assuming we get to keep the numbering currently in the TOC. Otherwise we'll have to change it.